marklogic-data-hub-central/src/test/resources/test-projects/run-flow-test/modules/root/custom-modules/custom/testStep.sjs

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+function main(content, options) {
+    const valueObj = content.value.toObject();
```

Comment:
Can you add a property to the step and confirm that it is available in 'options' in the module ?